modules/otel-troubleshoot-metrics.adoc

```diff
@@ -42,3 +40,19 @@ $ oc port-forward <collector_pod>
 ----
 
 . Access the metrics endpoint at `+http://localhost:8888/metrics+`.
+
+Alternatively, you can enable the creation of PodMonitors or ServiceMonitors (depending on the deployment mode of the OpenTelemetryCollector) to scrap the internal metrics by setting to `true` the `enableMetrics` setting.
+
++
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+spec:
+  mode: deployment
+  observability:
+    metrics:
+      enableMetrics: true # <1>
+----
+
+If `User Workload Monitoring` is enabled, you will be able to visualize the scraped metrics in the `Observe` section from the OpenShift Console. Note you can select to visualize the specific metrics from an OpenTelemetry Collector instance or OpenTelemetry component (processor, receiver or exporter).
```

Comment:
@iblancasa, what is `User Workload Monitoring`? Is this a field in a CR or a UI button somewhere?